lib/match-conditions.js

```diff
@@ -3,8 +3,12 @@
 var fs = require('fs');
 var execSync = require('child_process').execSync;
 var _ = require('lodash');
+var semver = require('semver');
 
 var version = process.version;
+var semVersion = semver.major(process.version) + '.' +
```

Comment:
if you use `process.versions.node` you can get the version number without the preceded `v`. This should allow us to avoid this bit of code. The way we are doing the search for versions we should be able to safely drop the `v` without any change in functionality.

Alternatively, if we would like to keep the `v`, we can still replace this line with `var semVersion = process.versions.node`